stablehlo/dialect/TypeInference.cpp

```diff
@@ -627,6 +627,18 @@ LogicalResult verifyReduceWindowOpInputsAndInferWindow(
   return success();
 }
 
+// Shape function can be called directly from autogenerated `build()` function,
+// which may not guarantee the added region(s) in `odsState.regions` to be
+// non-empty. Need check it here to avoid a crash for the ops that need regions
+// in type inference, like `IfOp/CaseOp/MapOp`.
+// An alternative way is to consider disable the autogenerated `build()` at all.
```

Comment:
Let's drop this sentence. We've made a decision to not go this way, so there's no need to talk about it in the code to avoid confusing the readers.

Reply:
Sure.